msrestazure/azure_cloud.py

```diff
@@ -114,14 +118,20 @@ class Cloud(object):  # pylint: disable=too-few-public-methods
     def __init__(self,
                  name,
                  endpoints=None,
-                 suffixes=None):
+                 suffixes=None,
+                 profile=None,
+                 is_active=False):
         self.name = name
         self.endpoints = endpoints or CloudEndpoints()
         self.suffixes = suffixes or CloudSuffixes()
+        self.profile = profile
```

Comment:
I truly think it's a mistake to have "profile" here. it has nothing to do with the static definition of the cloud, since it's a dynamic info.
Same for is_active.

I removed them on purpose when I port this code.